packages/react-cosmos-component-playground/src/components/FixtureList/dataMapper.js

```diff
@@ -10,6 +10,8 @@ const dataObjectToNestedArray = (base, path = '') => {
         name: key,
         expanded: true,
         children
+        // TODO: Enable this when we'll have component pages
+        // https://github.com/react-cosmos/react-cosmos/issues/314
```

Comment:
For this feature, we'll also want to make sure we don't accidentally copy over fixture from currentUrlParams when generating hrefs and handling clicks in the Tree component.